includes/admin/views/html-admin-page-status-logs-db.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Admin View: Page - Status Database Logs
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+?>
+<form method="get" id="mainform" action="">
+
+	<?php $log_table_list->views(); ?>
+	<?php $log_table_list->search_box( __( 'Search tag', 'woocommerce' ), 'tag' ); ?>
```

Comment:
Yes, `tag` is probably best avoided. `source` might be a good solution, it indicates that you should tell where your log comes from. Other generic taxonomy names might be `group`, `package`, `bundle`...